# openssl-1.0.2k made some trivial changes in the domd script that are
# not compatible with the version of /bin/sh on our Solaris 10 build
# machines. This reverts the changes to domd, effectively downgrading
# it to the version that was included with openssl-1.0.2j
diff -urN openssl-1.0.2k.orig/util/domd openssl-1.0.2k/util/domd
--- openssl-1.0.2k.orig/util/domd	2017-01-26 05:22:04.000000000 -0800
+++ openssl-1.0.2k/util/domd	2017-04-17 11:45:24.423400754 -0700
@@ -11,9 +11,7 @@
 fi
 if [ "$MAKEDEPEND" = "" ]; then MAKEDEPEND=makedepend; fi
 
-# Preserve Makefile timestamp by moving instead of copying (cp -p is GNU only)
-mv Makefile Makefile.save
-cp Makefile.save Makefile
+cp Makefile Makefile.save
 # fake the presence of Kerberos
 touch $TOP/krb5.h
 if ${MAKEDEPEND} --version 2>&1 | grep "clang" > /dev/null ||
@@ -34,12 +32,7 @@
     ${PERL} $TOP/util/clean-depend.pl < Makefile > Makefile.new
     RC=$?
 fi
-if ! cmp -s Makefile.save Makefile.new; then
-    mv Makefile.new Makefile
-else
-    mv Makefile.save Makefile
-    rm -f Makefile.new
-fi
+mv Makefile.new Makefile
 # unfake the presence of Kerberos
 rm $TOP/krb5.h
 
